src/components/DropZone/DropZone.tsx

```diff
@@ -22,21 +22,23 @@ import IconDragDrop from './icons/drag-drop.svg';
 import IconAlertCircle from './icons/alert-circle.svg';
 
 import {fileAccepted, getDataTransferFiles} from './utils';
-import {DropZoneContext} from './types';
+import {DropZoneContext, Size} from './types';
 
 import styles from './DropZone.scss';
 
 export type Type = 'file' | 'image';
 
 export interface State {
   id: string;
-  size: string;
+  height: Size;
+  width: Size;
   type?: string;
   error?: boolean;
   dragging: boolean;
   overlayText?: string;
   errorOverlayText?: string;
   numFiles: number;
+  loading: boolean;
```

Comment:
Is it possible to solve the flashing problem without adding state to the component?

Reply:
@solona what is the flashing problem?

Reply:
It's Dan's gif  [above](https://github.com/Shopify/polaris-react/pull/908#issuecomment-458548206).  Is there another example @solona I could take a look at?

Reply:
This measuring happens only initial load. So we could use componentDidMount and forceUpdate() on a private variable but since I think since this change only occurs once it's probably more performant as state.

I'd probably stick to `measuring` as oppose to `loading` for the state key, a little more clear. What do you think?